localstack/services/firehose/firehose_api.py

```diff
@@ -190,27 +225,21 @@ def put_records(stream_name: str, records: List[Dict]) -> Dict:
                 elif "data" in record:
                     data = base64.b64decode(record["data"])
 
-                body = json.loads(data)
+                try:
+                    body = json.loads(data)
+                except Exception as e:
+                    LOG.warning("Elasticsearch only allows json input data!")
+                    raise e
 
+                LOG.debug("Publishing to ES destination. Data: %s", data)
```

Comment:
nit: As documents may potentially become quite big, maybe we could use our `truncate(..)` util here for logging, e.g.:
```
LOG.debug("Publishing to ES destination. Data: %s", truncate(data, max_length=300))
```